Source/WebKit/UIProcess/gtk/AcceleratedBackingStore.cpp

```diff
@@ -42,11 +43,34 @@
 namespace WebKit {
 using namespace WebCore;
 
+#if PLATFORM(WAYLAND)
+static bool gtkCanUseHardwareAcceleration()
+{
+    static bool canUseHardwareAcceleration;
+    static std::once_flag onceFlag;
+    std::call_once(onceFlag, [&] {
```

Comment:
I would explicitly capture the boolean flag, to avoid accidental capture of other variables in scope.

```suggestion
    std::call_once(onceFlag, [&canUseHardwareAcceleration] {
```

Reply:
static variables can't be captured, but they don't need to be captured in any case.